### PR TITLE
Update inventory/create-customer.md template

### PR DESCRIPTION
**Pull request by ygyryn**
## Purpose of this pull request
This pull request (PR) fixing not correct configuration path for "Access Token Expiration" section

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/rest/tutorials/inventory/create-customer.html

## Links to Magento source code
/guides/v2.3/rest/tutorials/inventory/create-customer.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/rest/tutorials/inventory/create-customer.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ The customer `id` is `3`.
 
 To generate a customer's access token, you must specify the customer's username and password in the payload. You do not specify an admin {% glossarytooltip 34ecb0ab-b8a3-42d9-a728-0b893e8c0417 %}authorization{% endglossarytooltip %} token.
 
-By default, a customer token is valid for 1 hour. To change this value, click **Configuration** > **Services** > **OAuth** > **Access Token Expiration**. Then enter a new value for **Customer Token Lifetime (hours)**.
+By default, a customer token is valid for 1 hour. To change this value, click **Stores** > Settings > **Configuration** > **Services** > **OAuth** > **Access Token Expiration**. Then enter a new value for **Customer Token Lifetime (hours)**.
```
Comment (osrecio) [inline guides/v2.3/rest/tutorials/inventory/create-customer.md]:
```suggestion
By default, a customer token is valid for 1 hour. To change this value, click **Stores** > **Settings** > **Configuration** > **Services** > **OAuth** > **Access Token Expiration**. Then enter a new value for **Customer Token Lifetime (hours)**.
```

Comment (ygyryn) [reply guides/v2.3/rest/tutorials/inventory/create-customer.md]: Some maintainers are removing bolding for second point "Settings" , e.g:
https://github.com/magento/devdocs/pull/3969/commits/3c5b3b80284ef522abf3f01ca742d41114396994
That's why I try do the same.

Comment (dobooth) [reply guides/v2.3/rest/tutorials/inventory/create-customer.md]: Good. Only bold for things you have to click on.

**Pull request merged**